docs/yaml_docs/pipeline_preprocess_yml.md

```diff
@@ -0,0 +1,437 @@
+        Available options are: "signac", "logTF" or "logIDF".
+
+  - <span class="parameter">feature_selection_flavour</span> `String`, Default: signac<br>
+        Flavor for selecting highly variable features (HVF).
+        HVF selection either with scanpy's `pp.highly_variable_genes()` function or a `pseudo-FindTopFeatures()` function of the signac package.
+        Accordingly, available options are: "signac" or "scanpy".
+
+  - <span class="parameter">min_mean</span> `Float`, Default: 0.05<br>
+        Applicable if `feature_selection_flavour` is set to "scanpy".
+        You can leave this parameter blank if you want to use the default value.
+
+  - <span class="parameter">max_mean</span> `Float`, Default: 1.5<br>
+        Applicable if `feature_selection_flavour` is set to "scanpy".
+        You can leave this parameter blank if you want to use the default value.
+
+  - <span class="parameter">min_disp</span> `Float`, Default: 0.5<br>
+        Applicable if `feature_selection_flavour` is set to "scanpy".
+        You can leave this parameter blank if you want to use the default value.
+
+  - <span class="parameter">n_top_features</span> `Integer`<br>
+        Applicable if `feature_selection_flavour` is set to "scanpy".
+        Number of highly-variable features to keep.
+        If specified, overwrites previous defaults for HVF selection.
+
+  - <span class="parameter">filter_by_hvf</span> `Boolean`, Default: False<br>
+        Applicable if `feature_selection_flavour` is set to "scanpy".
+        Set to True if you want to filter the ATAC layer to retain only HVFs.
+
+  - <span class="parameter">min_cutoff</span> `String`, Default: q5<br>
+        Applicable if `feature_selection_flavour` is set to "signac".
+        Can be specified as follows:
+    - "q[x]": "q" followed by the minimum percentile, e.g. q5 will set the top 95% most common features as higly variable.
+    - "c[x]": "c" followed by a minimum cell count, e.g. c100 will set features present in > 100 cells as highly variable.
+    - "tc[x]": "tc" followed by a minimum total count, e.g. tc100 will set features with total counts > 100 as highly variable.
+    - "NULL": All features are assigned as highly variable.
+    - "NA": Highly variable features won't be changed.
+
+  - <span class="parameter">dimred</span> `String`, Default: LSI<br>
+        Available options are: PCA or LSI.
+        LSI will only be computed if TFIDF normalisation was used.
+
+  - <span class="parameter">n_comps</span> `Integer`, Default: 50<br>
+        Number of components to compute.
+
+  - <span class="parameter">solver</span> `String`, Default: default<br>
+        If using PCA, which solver to use. Setting this parameter to "default", will use the 'arpack' solver.
+
+  - <span class="parameter">color_by</span> `String`, Default: sample_id<br>
+        Specify the covariate you want to use to color the dimensionality reduction plot.
+
+  - <span class="parameter">dim_remove</span> `TODO`<br>
```

Comment:
I'm not sure in what format the `dim_remove` parameter should be provided. As a boolean? Or an integer?